Makefile

```diff
@@ -193,7 +199,7 @@ tf_apply_universal: ## Apply terraform
 .PHONY: tf_auto_apply
 tf_auto_apply: ## Auto apply terraform
 	make tf_set_workspace && \
-	terraform -chdir=./infrastructure/aws apply -auto-approve -var-file=$(CONFIG_DIR)/${env}-input-params.tfvars ${tf_build_args}
+	terraform -chdir=./infrastructure/aws apply -auto-approve -lock-timeout=300s -var-file=$(CONFIG_DIR)/${env}-input-params.tfvars -var=image_tag=$(IMAGE_TAG) ${tf_build_args} -target="module.django-app" -target="module.core-api" -target="module.worker"
```

Comment:
image_tag already gets defined in tf_build_args, so don't think this is necessary